pinion_adverts.sp

```diff
@@ -577,40 +607,57 @@ public OnPluginStart()
 #endif
 }
 
+public Action:FuncChatHook(Handle:event, const String:name[], bool:dontBroadcast)
+{
+	new client = GetClientOfUserId(GetEventInt(event, "userid"));
+	decl String:strChat[256]; 
+	GetEventString(event, "text", strChat, sizeof(strChat));
+
+	if (StrContains(strChat, "!RewardMe", false) == 0)
```

Comment:
@decrypted 
`if (StrContains(strChat, "!RewardMe", false) == 0)`
 Is a case insensitive search.